cli_reference/basic_cli_operations.adoc

```diff
@@ -136,32 +190,70 @@ One of the fundamental capabilities of OpenShift is the ability to build applica
 |Retrieves the build logs for the specified build.
 |===
 
-== Deployment CLI Operations
-OpenShift provides CLI access to inspect and manipulate deployment configurations using standard `osc` resource operations such as `get`, `create`, and `describe`.
-
-=== Deployment Configurations
+== Deployment Operations
+OpenShift provides CLI access to inspect and manipulate
+link:../dev_guide/deployments.html[deployment configurations] using standard
+`osc` resource operations such as `get`, `create`, and `describe`.
 
-The syntax to describe a deployment configuration in human-readable form is:
+To describe a deployment configuration in human-readable form:
 
 ****
-`osc deploymentConfigs describe [replaceable]#<deployment config>#`
+`$ osc describe dc _<deployment_config>_`
 ****
 
-=== Deployment Rollbacks
+For example:
 
-Rollbacks revert an application back to a previous deployment. Note that environment variables and volumes are included in rollbacks, so there are a variety of considerations to make when deciding whether a rollback is viable, such as:
+====
+
+[options="nowrap"]
+----
+$ osc describe dc docker-registry
+Name:		docker-registry
+Created:	18 hours ago
+Labels:		docker-registry=default
+Latest Version:	1
+Triggers:	Config
+Strategy:	Recreate
+Template:
+	Selector:	docker-registry=default
+	Replicas:	1
+	Containers:
+		NAME		IMAGE					ENV
+		registry	openshift/origin-docker-registry:v0.4.3	OPENSHIFT_CA_DATA=[omitted for space],OPENSHIFT_MASTER=https://10.245.2.2:8443
+Latest Deployment:
+	Name:		docker-registry-1
+	Status:		Complete
+	Selector:	deployment=docker-registry-1,deploymentconfig=docker-registry,docker-registry=default
+	Labels:		docker-registry=default
+	Replicas:	1 current / 1 desired
+	Pods Status:	1 Running / 0 Waiting / 0 Succeeded / 0 Failed
+----
+====
 
-* If security credentials have been recently updated, the previous deployment may not have the correct values.
-* If the previous deployment used a custom strategy, which is no longer available or usable, the deployment may not be deployed correctly.
+*Deployment Rollbacks*
 
-By default, only the pod and container configuration will be changed and scaling or trigger settings will be left as-is.
+Rollbacks revert an application back to a previous deployment. Note that
+environment variables and volumes are included in rollbacks, so there are a
+variety of considerations to make when deciding whether a rollback is viable,
+such as:
 
```

Comment:
[Suggestion]
Rollbacks revert an application back to a previous deployment, and they include environment variable and volumes. Therefore, when you decide whether a rollback is viable or not, among other things, consider the following: